config/templates/cloud-config-etcd

```diff
@@ -120,3 +132,11 @@ write_files:
   - path: /etc/etcd2/ssl/etcd.pem.enc
     encoding: gzip+base64
     content: {{.TLSConfig.EtcdCert}}
+
+  - path: /etc/etcd2/ssl/etcd-client.pem.enc
+    encoding: gzip+base64
+    content: {{.TLSConfig.EtcdClientCert}}
+
+  - path: /etc/etcd2/ssl/etcd-client-key.pem.enc
+    encoding: gzip+base64
+    content: {{.TLSConfig.EtcdClientKey}}
```

Comment:
Note to self:

* This would make user-data much bigger than before... Oh, we now have the `--s3-uri` option to workaround that. No problem 😃 
* This does add additional credentials we didn't have in etcd nodes before. Any security concerns for saving more credentials in etcd nodes? No.